klab.engine/src/main/java/org/integratedmodelling/klab/provenance/Artifact.java

```diff
@@ -247,7 +247,7 @@ public void setGenerator(Activity generator) {
 	 */
 	@Override
 	public boolean is(Class<?> cls) {
-		return peers.get(cls) != null;

```

Comment:
These changes, and the ones at `as()` are not just random whitespaces. At an intermediate commit, I added some logic here that was later moved to `DirectObservation`.